### PR TITLE
Service details sidecar improvements

### PR DESCRIPTION
**Pull request by renemennab**
## Summary
- adds namespace, repository and helmRepository to service details sidecar 
- restricts helm values to admin users

[linear issue](https://linear.app/pluralsh/issue/ENG-1945/service-details-sidecar-improvements)

### CODE REVIEW

File: assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx [hunk 1]
```diff
@@ -92,6 +107,11 @@ export function ServiceDetailsSidecar({
             <Chip severity="success">None</Chip>
           )}
         </SidecarItem>
+        {helmRepository && (
+          <SidecarItem heading="Helm Repository">
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx:111]: This should ideally include the helm icon like in the service table row

File: assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx [hunk 2]
```diff
@@ -103,6 +123,9 @@ export function ServiceDetailsSidecar({
             {helm.version}
           </SidecarItem>
         )}
+        {repository && (
+          <SidecarItem heading="Git repository">{repository.url}</SidecarItem>
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx:127]: same, this should ideally include a git icon

File: assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx [hunk 3]
```diff
@@ -37,7 +37,19 @@ export function ServiceDetailsSidecar({
...
+    status,
+    cluster,
+    git,
+    helm,
+    namespace,
+    repository,
+    helmRepository,
+  } = serviceDeployment
+
+  console.log('repository', repository)
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx:52]: rm

File: assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx (hunk 1 above)
Comment (renemennab) [reply assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx:113]: I added the icons, I am just not sure if they are in the right place. I aligned them with the urls, is that ok?

Comment (michaeljguarino) [reply assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx:113]: yeah they should be horizontally aligned w/ the urls

File: assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx [hunk 4]
```diff
@@ -103,6 +139,24 @@ export function ServiceDetailsSidecar({
             {helm.version}
           </SidecarItem>
         )}
+        {repository && (
+          <SidecarItem heading="Git repository">
+            <div
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx:144]: Is there not some reusable component for this already?  If there isn't we probably should make one and put it in the utils library, can't imagine we won't need more icon-ed strings in the future.

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**